### DISCUSSION TITLE
插件今天发新版播放器也新版，请问有没有办法暂停 b 站播放器更新，不然插件老是用不了。

### DISCUSSION THREAD

**Discussion by smart-z** [the1812/Bilibili-Evolved#3323] [提问 / Q&A] [answered]
今天 b 站播放器更了 3.8.14 ，脚本刚好没支持到。播放器窗口那么小，每次都要手动点一下网页全屏。真 tm 操蛋。

抱歉我不是程序员，对 github 也不太了解，如果提问有问题，真的非常抱歉了。求指教求帮助。谢谢。

**Comment by Kal-tsit63** [comment]
具体对应哪个功能？

**Reply by smart-z** [reply]
默认播放模式
我个人最忍受不了的一个就是“默认播放模式”。网页全屏没有快捷键，默认播放尺寸非常小，每次都要点网页全屏或者宽屏。其他的倒还好，忍忍。😭

**Comment by Geeyun-JY3** [comment, marked as answer]
灰度的事没办法的：#3187

**Reply by smart-z** [reply]
看到了，谢谢
